app/controllers/admin/reports_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Admin
+  class ReportsController < ApplicationController
```

Comment:
no need for concern here. create a new base_controller in admin namespace.
`Admin::BaseController` and then inherit ReportsController from that